### PR TITLE
Doc/add regressor comparison tuto

### PR DESCRIPTION
**Pull request by AmT42**
adapte the regressor tutorial  for concrete 1.0 and make the PR on right branch and repo

### CODE REVIEW

**Comment by aquint-zama** [comment]
relates to https://github.com/zama-ai/bounty-program/issues/14

File: docs/advanced_examples/ReggresorComparison.ipynb [hunk 1]
```diff
@@ -0,0 +1,514 @@
+{
+ "cells": [
+  {
```
Comment (bcm-at-zama) [inline docs/advanced_examples/ReggresorComparison.ipynb:3]: Hey. The filename "docs/advanced_examples/ReggresorComparison.ipynb" has typos. That's "regressor", with a single "g" and two "s".

Comment (AmT42) [reply docs/advanced_examples/ReggresorComparison.ipynb]: Inatention error, it's rectified thank you

**Comment by jfrery** [comment]
Hello @AmT42  and thanks again for your work! We would like to proceed to the merge. Could you rebase on the https://github.com/zama-ai/concrete-ml/tree/tutorials branch please?

**Comment by aquint-zama** [comment]
@cla-bot check

**Comment by jfrery** [comment]
Thanks again for your work here. We are merging your commit with the tutorial branch and we will include it in the next release of CML.

**Comment by aquint-zama** [comment]
close https://github.com/zama-ai/bounty-program/issues/41

**Pull request merged**